debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -527,6 +527,25 @@ public void shouldReceiveChangesForInsertsIndependentOfReplicaIdentity() throws
         assertInsert(statement, 5, Collections.singletonList(new SchemaAndValueField("text", SchemaBuilder.OPTIONAL_STRING_SCHEMA, "no_pk_and_default")));
     }
 
+    @Test
+    public void shouldNotStreamAllOperationsWithSkippedOperations() throws Exception {
+        TestHelper.execute("CREATE TABLE public.skipped_op_table (pk SERIAL, data varchar(25) NOT NULL, PRIMARY KEY (pk));");
+        startConnector(config -> config
+                .with(PostgresConnectorConfig.SKIPPED_OPERATIONS, "d")
+                .with(PostgresConnectorConfig.SNAPSHOT_MODE, SnapshotMode.NEVER),
+                false);
+        consumer = testConsumer(0);
+
+        // Insert
+        TestHelper.execute("INSERT INTO skipped_op_table (data) VALUES ('test')");
+        consumer.expects(1);
+
+        // Delete
```

Comment:
It's important to assert on a nother non-skipped record after that. I'd suggest to simply follow the same test flow I used for MySQL.

Reply:
Thanks. I'm going to do that